xml/ha_maintenance.xml

```diff
@@ -0,0 +1,375 @@
+  </para>
+
+  <para>
+   If you attempt to start &pace; services on a node while the cluster is in
+   maintenance mode, &pace; will initiate a single one-shot monitor operation
+   (a <quote>probe</quote>) for every resource to evaluate which resources are
+   currently running on that node. However, it will take no further action
+   other than determining the resources' status.
+  </para>
+ </sect1>
+ <sect1 xml:id="sec.ha.maintenance.outline">
+  <title>Avoiding Unwanted Side Effects</title>
+  &warning-maint-mode;
+</sect1>
+ <sect1 xml:id="sec.ha.config.hawk.maint.mode">
+  <title>Using Maintenance Modes with &hawk2;</title>
+
+  <para>
+   The following procedures show how to set and unset maintenance mode on
+   cluster level, node level, and resource level with &hawk2;. If you want to
+   put a resource into unmanaged mode with the <option>is-managed</option> meta
+   attribute, see <xref linkend="pro.ha.config.hawk.rsc.is-managed"/>.
+  </para>
+
+  <procedure xml:id="pro.ha.config.hawk.maint.mode.cluster">
+   <title>Applying Maintenance Mode to the Cluster</title>
+   <step>
+    <para>
+     Start a Web browser and log in to the cluster as described in
+     <xref linkend="sec.conf.hawk2.login"/>.
+    </para>
+   </step>
+   <step>
+    <para>
+     In the left navigation bar, select <guimenu>Cluster
+     Configuration</guimenu>.
+    </para>
+   </step>
+   <step>
+    <para>
+     In the <guimenu>CRM Configuration</guimenu> group, select the
+     <guimenu>maintenance-mode</guimenu> attribute from the empty drop-down box
+     and click the plus icon to add it.
+    </para>
+   </step>
+   <step>
+    <para>
+     To set <literal>maintenance-mode=true</literal>, activate the check box
+     next to <literal>maintenance-mode</literal> and confirm your changes.
+    </para>
+   </step>
+   <step>
+    <para>
+     After you have finished the maintenance task for the whole cluster,
+     deactivate the check box next to the <literal>maintenance-mode</literal>
+     attribute.
+    </para>
+    <para>
+     From this point on, &hasi; will take over cluster management again.
+    </para>
+   </step>
+  </procedure>
+
+  <procedure xml:id="pro.ha.config.hawk.maint.mode.nodes">
+   <title>Applying Maintenance Mode to Nodes</title>
+   <step>
+    <para>
+     Start a Web browser and log in to the cluster as described in
+     <xref linkend="sec.conf.hawk2.login"/>.
+    </para>
+   </step>
+   <step>
+    <para>
+     In the left navigation bar, select <guimenu>Cluster Status</guimenu>.
+    </para>
+   </step>
+   <step>
+    <para>
+     In one of the individual nodes' views, click the wrench icon next to the
+     node and select <guimenu>Maintenance</guimenu>.
+    </para>
+    <para>
+     This will add the following instance attribute to the node:
+     <literal>maintenance="true"</literal>. The resources previously running on
+     the maintenance-mode node will become <literal>unmanaged</literal>. No new
```

Comment:
How about "The resources previously running on this node will now become "unmanaged."?

Reply:
Done.